src/styles/main.scss

```diff
@@ -0,0 +1,102 @@
+body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+.box {
+  position: relative;
+
+  &__hour {
+    width: 10px;
+    height: 300px;
+    top: -300px;
+    background-color: red;
+    position: absolute;
+    transform-origin: bottom;
+    transform: rotate(0.01turn);
+    -webkit-animation-duration: 7s;
+    animation-duration: 7s;
+    -webkit-animation-name: move;
+    animation-name: move;
+  }
+
+  &__minute {
+    width: 10px;
+    height: 200px;
+    top: -200px;
+    background-color: #4b9aaa;
+    position: absolute;
+    transform-origin: bottom;
+    transform: rotate(0.01turn);
+    -webkit-animation-duration: 3s;
+    animation-duration: 3s;
+    -webkit-animation-name: move;
+    animation-name: move;
+  }
```

Comment:
there are a lot of repeatable styles applied both for hour hand and minute one. Either use mixin/placeholder or a common class with repeatable styles